developer_guides/debugability/logger/index.rst

```diff
@@ -204,19 +204,27 @@ Instance descriptions can have one of the following forms:
 - ``X.*`` - each component on selected pipeline *X*
 - ``X.Y`` - component on pipeline *X* with id *Y*
 
-Trace level changes works in the same order as options given in a command line, and a new set overwrites old values. It allows you to easily enable verbose logs only for selected components and keep the lowest possible log level (critical) for others, as shown in the following example:
+Trace level changes work in the same order as options given in a command line, and a new set overwrites old values.
+It allows you to easily enable verbose logs only for selected components and keep the lowest possible log level (critical) for
+others, as shown in the following example:
 
    sof-logger -l ldc_file -t -Fcritical=* -Fverbose="dai*, volume1.1"
 
 A similar example may be prepared for components on a particular pipeline:
 
-   sof-loggerr -l ldc_file -t -Fc=* -Fv=*1.*
+   sof-logger -l ldc_file -t -Fc=* -Fv=*1.*
 
-.. note::
-  To track a verbose message, select the "Trace verbose" option under the "Trace" menu from the firmware build.
+Verbose and debug log levels
+----------------------------
+
+To enable verbose and debug trace messages, select the "Trace->Trace verbose" option in the firmware build
+menuconfig (in addition to setting the proper log levels as described above).
```

Comment:
This still needs separate commits. One commit that does the rephrasing/separation of info, and one that fixes whitespace/typos/etc. Also the commit message being in all lowercase isn't ideal.